Tasks/DeployVisualStudioTestAgent/task.json

```diff
@@ -52,7 +52,7 @@
             "defaultValue": "",
             "required": false,
             "helpMarkDown": "Comma separated list of machines where test agent needs to be deployed. For example: 'Machine1,Machine2,Machine3'",
-            "groupName": "agentConfiguration"
+            "groupName": "environment"
```

Comment:
though this is just the group name and not seen in ui anywhere but because the code is open source we need to fix the naming.